Resources/views/CRUD/base_list.html.twig

```diff
@@ -295,7 +295,13 @@ file that was distributed with this source code.
                         </div>
 
                         {% for paramKey, paramValue in admin.persistentParameters %}
-                            <input type="hidden" name="{{ paramKey }}" value="{{ paramValue }}">
+                            {% if paramValue is iterable %}
+                                {% for key, value in paramValue %}
+                                    <input type="hidden" name="{{ paramKey }}[{{ key }}]" value="{{ value }}">
+                                {% endfor %}
```

Comment:
@OskarStark I got a version which support an undefined level of arrays with a recursive twig macro.

What did you prefer? The support of one level with simple code or unlimited levels with a more complex code (you can see it below).

``` twig
{% for paramKey, paramValue in admin.persistentParameters %}
	{{ _self.hidden_input(paramKey, paramValue, []) }}
{% endfor %}

{% macro hidden_input(paramKey, paramValue, chain) %}
	{% if paramValue is iterable %}
		{% for key, value in paramValue %}
			{% set chain = chain|merge([key]) %}
			{{ _self.hidden_input(paramKey, value, chain) }}
		{% endfor %}
	{% else %}
		{% set name = paramKey %}
		
		{% for level in chain %}
			{% set name = name ~ "[" ~ level ~ "]" %}
		{% endfor %}
		
		<input type="hidden" name="{{ name }}" value="{{ paramValue }}">
	{% endif %}
{% endmacro %}
```